releasenotes/notes/add-builder-command-5ae301b6e0f9d6bd.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
```

Comment:
Please remove this note because this is now change in the internals (the builder class is protected) so there is no need for end user communication.

Reply:
I fixed at https://github.com/Qiskit/qiskit-terra/pull/10253/commits/8effc991873662ea93037e2b70803776b5d8568f.